### ISSUE TITLE
Safe Browsing setting is potentially misleading

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#5125]
The Safe Browsing setting in `chrome://settings/privacy` says that some unsafe pages might be sent to Brave:

![Screenshot from 2019-07-03 23-17-36](https://user-images.githubusercontent.com/167821/60643736-e61acb80-9de8-11e9-922f-b9bc892e5d27.png)

I'm not 100% sure what part of Safe Browsing it's referring to, but I doubt it's sending Brave URLs. It's most likely sending these to nobody (because we turned it off) or it's sending these to Google under some limited circumstances.

cc @jumde

**Comment by bsclifton** [comment]
@fmarier these URLs should be sent to us and then proxied to Google. We have a Google Cloud platform account which has a token used to handle this service

**Comment by fmarier** [comment]
Yes, we do proxy Safe Browsing requests. In the case of browsing protection (e.g. the phishing warning pages), that includes both updates and requests for full hashes, but those [are not actual URLs](https://feeding.cloud.geek.nz/posts/how-safe-browsing-works-in-firefox/).

On the other hand, for download protection (i.e. how downloads are checked for maliciousness with Google Safe Browsing) we do currently send URLs to Google (to be disabled in #4341).

There are other parts of Safe Browsing which use URLs and which I need to investigate (and disable if they're active): client-side phishing detection), extended_reporting and password_protection.

While we proxy all of the requests, we're not looking at them, only Google is. So it seems misleading to state that some URLs might get sent to Brave, when they are really sent to Google _via_ Brave.

**Comment by tomlowenthal** [comment]
@fmarier Do we have access to the plaintext of either the hash-prefix requests or the URL-based requests as they transit our proxy?

**Comment by fmarier** [comment]
Since we are using a `brave.com` endpoint for the requests, I believe we have access to the hash prefixes and the soon-to-be-disabled URLs in download protection requests.

**Comment by rebron** [comment]
![Screen Shot 2020-06-16 at 10 22 30 AM](https://user-images.githubusercontent.com/5951041/84806942-57806980-afbb-11ea-8568-3dcb0060bab1.png)

I believe this was changed a couple versions ago but in 1.10.90 Chromium: 83.0.4103.97 that's the current text. 

Are we ok to close this one out or do we need additional changes?

**Comment by fmarier** [comment]
That message was changed to solve a different problem (i.e. the need to credit Google).

While we don't look at the traffic we proxy and immediately strip out the client IP address, the issue that we could see some of the traffic that we proxy remains. The description doesn't convey that, but perhaps it's already adequately covered in our privacy policy.

**Comment by rebron** [comment]
@fmarier 
<img width="684" alt="Screen Shot 2021-10-06 at 9 04 16 AM" src="https://user-images.githubusercontent.com/5951041/136241311-585cad1d-39a1-4835-86a3-bb51837423ee.png">

Text has changed. Stale issue and ok to close or do you have recommendations to update the current text?
